lmdeploy/pytorch/kernels/cuda/pagedattention.py

```diff
@@ -38,7 +35,9 @@ def tanh(x):
     triton.Config({}, num_stages=2, num_warps=8),
     triton.Config({}, num_stages=2, num_warps=4),
 ],
-                 key=['BLOCK_H', 'BLOCK_N', 'BLOCK_DMODEL', 'BLOCK_DV'])
+                 key=['BLOCK_H', 'BLOCK_N', 'BLOCK_DMODEL', 'BLOCK_DV'],
+                 warmup=10,
+                 rep=25)
```

Comment:
what does `rep` mean?

Reply:
repeat benchmark for autotuning.

Reply:
How long does this setting `warmup=10, rep=25`  take?

Reply:
each tuning config take 34~35ms